scripts/coverage

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+TIX_FILE="$(stack path --local-hpc-root)/combined/custom/custom.tix"
+LCOV_FILE="./coverage/lcov.info"
+
+mkdir ./coverage 2> /dev/null;
+stack exec -- hpc-lcov --file $TIX_FILE -o $LCOV_FILE --main-package madlib;
+lcov -r coverage/lcov.info '.stack-work/*' -o coverage/lcov.info;
```

Comment:
So you don't miss it @brekk. Here I found a simpler way to skip grammar's coverage for now. Mostly because it seems to generate a lot of stuff that is not reached at all, like huge functions where not even a branch is covered. Well, we had that talk already. Mostly I think the grammar coverage isn't relevant as is, and we can mainly test it through compilation and type checking.